relation_engine_server/utils/json_validation.py

```diff
@@ -73,9 +68,9 @@ def get_schema_validator(schema=None, schema_file=None, validate_at=""):
     validation_schema = resolve_pointer(schema, validate_at)
 
     if schema_file:
-        resolver = ExtendedRefResolver(schema_file, schema)
+        resolver = RefResolver(schema_file, schema)
     else:
-        resolver = ExtendedRefResolver.from_schema(schema)
+        resolver = RefResolver.from_schema(schema)
```

Comment:
does the updated version of the `jsonschema` lib now parse `yaml` files as well as `json`?

Reply:
I don't think so ... I only updated `jsonschema` trying to fix that obtuse bug where the `jsonschema` error message was not helpful. In order to run things though, I had to get rid of some no-longer-existing `from jsonschema.compat import` things that were being used in `ExtendedRefResolver`, which I found out wasn't being used anywhere, serendipitously. So I axed `ExtendedRefResolver`. I could downgrade `jsonschema` again now that the bug was figured out, but I figure why not just keep things updated ...

Reply:
We should check that everything still works correctly without the ExtendedRefResolver -- it was put in place because the jsonschema package only parses JSON files, and a number of our conf files are in yaml (as you will know!). IIRC the ExtendedRefResolver is a fairly simple override of the existing jsonschema RefResolver with two additions: 1) assume that if there's no scheme for a ref, it's a local file, and 2) if the file name ends with '.yml' or '.yaml', use the yaml parser to open it.

See https://python-jsonschema.readthedocs.io/en/latest/_modules/jsonschema/validators/#RefResolver.resolve_remote